paper-progress.html

```diff
@@ -298,9 +298,13 @@
     },
 
     _toggleIndeterminate: function(indeterminate) {
+      this.indeterminate = indeterminate;
```

Comment:
`_toggleIndeterminate` is an observer (see line 278), so `this.indeterminate` is already set by the time this runs (and should not be set again.